.github/workflows/release-please.yml

```
@@ -0,0 +1,36 @@
+name: Release
+on:
+  push:
+    branches:
+      - 'ci/migrate-to-release-please'
+
+jobs:
+  release:
+    name: Create release
+    permissions:
+      contents: write
+      pull-requests: write
```

MUST: We need to specify `outputs` here to use `${{ needs.release.outputs.release_created }}` on publish job.

```suggestion
      pull-requests: write
    outputs:
      release_created: ${{ steps.release.outputs.release_created }}
```